oneapi/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+source /opt/intel/inteloneapi/setvars.sh 
+exec "$@"
```

Comment:
No newline at end of the file has caused me some very odd problems in the past.

Reply:
Fixed.